Docs/api-images.md

```diff
@@ -84,3 +84,28 @@ Fields description can be found [here](api-images-models.md).
 - `400` - request data didn't pass validation
 - `401` - missing JWT token
 - `403` - missing required permissions
+
+
+## POST: [api/images/tsv](http://localhost:5102/api/images/tsv) - [api/images-feed/tsv](https://localhost/api/images-feed/tsv)
+Submit images data (excluding image analysis data).
```

Comment:
Could we write here: "Submit images data in tsv format (excluding images analysis data)"